scripts/install-hpegl-provider-macos.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -e
+
+os="darwin"
+
+if [[ `uname -m` == 'x86_64' ]] 
+then
+  echo 'Intel Architecture detected'
+  arch="amd64"
+elif [[ `uname -m` == 'arm64' ]] 
+then
+  echo 'Apple Silicon Architecture detected'
+  arch="arm64"
+fi
+
+repo="HPE/terraform-provider-hpegl"
+macos_hpegl_dir="${HOME}/.terraform.d/plugins/local/hpe/hpegl"
```

Comment:
That's not going to work I'm almost certain.  You need `registry.hashicorp.io/hpe/hpegl`

Reply:
Ok will revert to what the original script uses - registry.terraform.io. This works for me.